packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/AppConfiguration.kt

```diff
@@ -149,6 +168,8 @@ public interface AppConfiguration {
         private var httpLogObfuscator: HttpLogObfuscator? = LogObfuscatorImpl
         private val customRequestHeaders = mutableMapOf<String, String>()
         private var authorizationHeaderName: String = DEFAULT_AUTHORIZATION_HEADER_NAME
+        private var enableSessionMultiplexing: Boolean = true
```

Comment:
This should be `false`. The feature is not tested extensively at SDK level to push it to production in this PR

Reply:
This has been reverted.